packages/icons/scripts/web.ts

```diff
@@ -63,24 +64,17 @@ function buildComponentFile(component: Component): string {
     name: getComponentName(`${icon.name}-${icon.size}`),
   }));
 
-  if (icons.length === 1) {
```

Comment:
Yeah this made sense when we had either 1 or 2 sizes, but I like your solution of simplifying this by just returning a single template here